src/templates/pix-download.html.php

```diff
@@ -0,0 +1,131 @@
+<?php
+if (!defined('ABSPATH')) {
+    exit;
+}
+?>
+<?php if (isset($vindi_order)) : ?>
+    <div class="vindi_payment_listing">
+        <div class="info_message">
+            <div class="icon"></div>
+            <div class="message">
+                <p class="message_title">
+                    <?php _e('Aqui estão as sua cobranças PIX.', VINDI); ?>
+                </p>
+                <p class="message_description">
+                    <?php
+                        _e(
+                            'Você pode pagar lendo o QR Code abaixo ou copiando a linha digitável. 
+                            Após recebermos a confirmação do pagamento, seu pedido será processado.',
+                            VINDI
+                        );
+                    ?>
+                </p>
+            </div>
+        </div>
+        <div class="charges">
+    <?php foreach ($order_to_iterate as $key => $subscription) : ?>
+                <?php if (is_array($subscription) && array_key_exists('product', $subscription)
+                        && !in_array($subscription['bill']['status'], array('paid', 'canceled'))) : ?>
+                        <div class="pix_charge charge">
+                            <span class="pix_product_title product_title">
+                                <?php echo $subscription['product']; ?>
+                            </span>
+                            <div>
+                                <?php if (new DateTime($subscription['bill']['pix_expiration']) < new DateTime()
```

Comment:
acredito que essa comparação deveria ser feita com o `new DateTime()` forçado para timezone GMT-3